### PR TITLE
zboss: version v3.11.4.0+5.1.5

### PR DESCRIPTION
**Pull request by AdamZelikNS**
- ZBOSS sources: 82b8031a171fa996b390540ed844498e63cea75a
- major modification: NVRAM improvements

### CODE REVIEW

**Comment by AdamZelikNS** [comment]
To do:
- [ ] update CHANGELOG.rst

File: zboss/CHANGELOG.rst [hunk 1]
```diff
@@ -9,6 +9,20 @@ Changelog
...
 All notable changes to this project in the |NCS| are documented in this file.
 
+Main branch
+***********
+
+Updated
+=======
+
+* The ZBOSS stack to version ``3.11.4.0+5.1.5``.
+  For detailed information, see `ZBOSS stack release notes`_ for the |NCS| v2.6.0 release.
```
Comment (edmont) [inline zboss/CHANGELOG.rst:19]:
```suggestion
  For detailed information, see `ZBOSS stack release notes`_ for the |NCS| v2.6.1 release.
```

**Review by edmont — APPROVED** [review]

**Review by wiba-nordic — APPROVED** [review]

**Comment by AdamZelikNS** [comment]
updated using zboss/*lib.a with pro_bv-13 fix and rebased

**Review by milewr — APPROVED** [review]

**Comment by AdamZelikNS** [comment]
a problem was encountered with nightly/test_factory_reset, therefore the DNM has been set, investigation in progress ..

**Comment by AdamZelikNS** [comment]
nightly/test_factory_reset fix has been applied to sdk-nrf (commit in the same pr as manifest)
now, here only rebasing done

I'm not removing the DNM now, I'll wait for more complete testing

**Comment by AdamZelikNS** [comment]
libraries updated, now built with ncs which includes sysbuild introducing changes

**Pull request merged**